django/db/migrations/operations/models.py

```diff
@@ -43,6 +53,11 @@ def __init__(self, name, fields, options=None, bases=None, managers=None):
         self.bases = bases or (models.Model,)
         self.managers = managers or []
         super(CreateModel, self).__init__(name)
+        # Sanity-check that there are no duplicated field names, bases, or
+        # manager names
+        check_for_duplicates((name for name, _ in self.fields), "field")
+        check_for_duplicates(self.bases, "base")
```

Comment:
Check against `(base._meta.label_lower if isinstance(base, models.base.ModelBase) else base.lower() for base in self.bases)` to catch mixed types bases duplicates (e.g. `bases=(MyModel, MyModel._meta.label_lower)`)